models/series/PieSeries.rst

```diff
@@ -30,4 +30,105 @@ Example
 
 .. sourcecode:: csharp
 
-    var model = new PlotModel { Title = "PieSeries" };
+    namespace ExampleLibrary
```

Comment:
I think line 33-58 should not be included in the example. It is inconsistent with the VB example, and will not work with the XAML below. Please fix :-)